### DISCUSSION TITLE
Controller freezing randomly

### DISCUSSION THREAD

**Discussion by AmoghBhagwat** [cyberbotics/wrestling#112] [General]
@omichel 
The controller of our bot freezes randomly, and does not start again for the remainder of the match. Is anyone else facing this issue?

**Comment by AmoghBhagwat** [comment]
@omichel and others, please have a look at [this](https://drive.google.com/file/d/1TocIS9sH58pDB3b4g8XWwOHAnkC7sI1_/view?usp=sharing), both the controllers do random actions at around 20s mark and our controller freezes, whereas the other one still does random actions. Anyone else facing this issue?

**Comment by kibergrad** [comment]
Yes @omichel , something similar is happening with my controller as well.
The controller was working perfectly yesterday, doing everything as per the code but somehow the same controller is having a lot of noise today.

**Comment by omichel** [comment]
Yes, I acknowledge the problem. I already investigated it without success so far. Unfortunately, it is too late to fix it now as we will be running the final games very soon. It seems however that this bug appears randomly. So, I will monitor the final games and whenever I suspect the bug to show up, I will stop the simulation and re-run it until the bug doesn't show-up anymore. Hopefully, this will work.

**Reply by kibergrad** [reply]
Yes, this will help removing the randomness.
Thanks.
